tools-core/build.gradle.kts

```diff
@@ -0,0 +1,40 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ */
+
+plugins {
+    id("com.hedera.hashgraph.java-conventions")
+    id("org.openjfx.javafxplugin") version "0.0.13"
+}
+
+dependencies {
+    implementation(libs.bundles.logging)
+    implementation(libs.bundles.bouncycastle)
+    implementation(libs.jackson.databind)
+    implementation(libs.eddsa)
+    implementation(libs.protobuf.javalite)
+    implementation(libs.protobuf.java.util)
+    implementation(libs.commons.codec)
+    implementation(libs.commons.io)
+    implementation(libs.httpclient)
+    implementation(libs.zt.zip)
+    implementation(libs.opencsv)
```

Comment:
I assume that we can define some of the dependencies as api (see https://docs.gradle.org/current/userguide/java_library_plugin.html#sec:java_library_configurations_graph)